WasatchNET/EEPROM.cs

```diff
@@ -295,6 +295,34 @@ public float[] wavecalCoeffs
 
         float[] _wavecalCoeffs;
 
+
+        public float[] intensityCorrectionCoeffs
+        {
+            get { return _intensityCorrectionCoeffs; }
+            set
+            {
+                EventHandler handler = EEPROMChanged;
+                _intensityCorrectionCoeffs = value;
+                handler?.Invoke(this, new EventArgs());
+            }
+        }
+
+
+        float[] _intensityCorrectionCoeffs;
+
+        public byte intensityCorrectionOrder
+        {
+            get { return _intensityCorrectionOrder; }
+            set
+            {
+                EventHandler handler = EEPROMChanged;
+                _intensityCorrectionOrder = value;
+                handler?.Invoke(this, new EventArgs());

```

Comment:
I might log a warning if set to a value other than _intensityCorrectionCoeffs.length - 1